interfaces/IFlight.ts

```diff
@@ -1,9 +1,13 @@
 export default interface IFlight {
+  id?: number;
```

Comment:
id is optional? That's weird. Why?

Reply:
took it directly from Daniele's website interface